client/my-sites/earn/components/commission-fees.tsx

```diff
@@ -18,16 +22,27 @@ const CommissionFees = ( {
 	isPlan100YearPlan,
 }: CommissionFeesProps ) => {
 	const translate = useTranslate();
+	const site = useSelector( getSelectedSite );
+	const isJetpack = useSelector( ( state ) => isJetpackSite( state, site?.ID ) );
+	const isAtomicSite = useSelector( ( state ) => isSiteAutomatedTransfer( state, site?.ID ) );
 
 	if ( commission === null ) {
 		return null;
 	}
 
+	const isJetpackNotAtomic = isJetpack && ! isAtomicSite;
+
+	const upgradeLinkHost = isJetpackNotAtomic ? 'https://jetpack.com' : 'https://wordpress.com';
+
 	const upgradeLink =
 		commission === 0 || isPlan100YearPlan ? null : (
 			<>
 				{ ' ' }
-				<a href={ siteSlug ? `/plans/${ siteSlug }` : '/plans' }>
+				<a
+					href={
+						siteSlug ? `${ upgradeLinkHost }/plans/${ siteSlug }` : `${ upgradeLinkHost }/plans`
```

Comment:
This breaks for me on a Jetpack self hosted site, for example I had a site: `shadowy-post-tree.jurassic.ninja` so it links to https://jetpack.com/plans/shadowy-post-tree.jurassic.ninja which shows Page not found.

![404](https://github.com/Automattic/wp-calypso/assets/6586048/9d2fe796-52db-4bc9-9e3f-84e3a86276a5)

Reply:
Thanks for noticing this. I fixed it, but we are waiting for an answer from here: https://github.com/Automattic/wp-calypso/pull/88778#issuecomment-2024819919